data/menus.yaml

```
@@ -23,7 +23,8 @@
 - title: Documentation
   items:
     - Learning resources (start here): learn.html
-    - API documentation: https://leanprover-community.github.io/mathlib4_docs
+    - General documentation: https://leanprover-community.github.io/mathlib4_docs
+    - Definition search (Loogle): https://loogle.lean-lang.org/
```

Loogle searches theorems too, so maybe

```suggestion
    - Semantic search (Loogle): https://loogle.lean-lang.org/
```